packages/tests/wdio.conf.js

```diff
@@ -221,8 +231,39 @@ exports.config = {
     /**
      * Function to be executed before a test (in Mocha/Jasmine) starts.
      */
-    // beforeTest: function (test, context) {
-    // },
+    beforeTest: function (test, context) {
+        // Override console to allow for STDOUT and file logging with one command
+        originalConsole = console
+
+        let filename = test['fullName'].replace(/\s/gi, '_');
+        let file = `${logsPath}/${test['start']}-${filename}.log`;
+
+        const logger = Logger.new(file);
+
+        console = (function(originalConsole){
+            return {
+                log: function(text){
+                    originalConsole.log(text);
+                    logger.info(text)
```

Comment:
It's not `logger.log` here?

Reply:
log4js doesn't have `.log`

console.log is an info message
https://developers.google.com/web/tools/chrome-devtools/console/log
> Each console method is assigned a severity level: Verbose, Info, Warning, or Error. For example, console.log() is an Info-level message